lib/generate-banner.js

```diff
@@ -1,4 +1,8 @@
-define(['lodash', 'package'], function(_, pkg) {
+define(['lodash', 'package', 'lib/build-hash'], function(_, pkg, buildHash) {
+  var domain = 'modernizr.com';
+  if ( typeof location !== 'undefined' && 'host' in location ) {
```

Comment:
jshint is gonna have a problem with this, gotta add a comment or a .jshintrc rule